pkgs/os-specific/linux/linux-wifi-hotspot/default.nix

```diff
@@ -44,6 +45,8 @@ stdenv.mkDerivation rec {
     substituteInPlace ./src/scripts/create_ap.service \
       --replace "/usr/bin/create_ap" "$out/bin/create_cap" \
       --replace "/etc/create_ap.conf" "$out/etc/create_cap.conf"
```

Comment:
In case I would like to still package upstream systemd unit I'll have to correct the path to `create_ap` script and config file. This systemd unit should still work with the default upstream configuration. Using the module you'll be able to generate your own config.